src/storage/DataAccessorBasedStore.ts

```diff
@@ -97,6 +123,21 @@ export class DataAccessorBasedStore implements ResourceStore {
   public async getRepresentation(identifier: ResourceIdentifier): Promise<Representation> {
     this.validateIdentifier(identifier);
 
+    if (this.metadataStrategy.isAuxiliaryIdentifier(identifier)) {
```

Comment:
The triple blocks might not make sense actually or you'll at least need a separate block where containers are handled since that is where there is a lot of overlap between metadata and normal request.